thirdparty/rabbitmq-server/rabbitmq-env.conf

```diff
@@ -0,0 +1,7 @@
+NODENAME=rabbit@localhost
```

Comment:
good we are fixing bug related to numerical hostname
please ask QE to validate this on multinode setup

Reply:
Sure